Failed to build "release/1.9.x" branch

```
github.com/hashicorp/consul/agent
# github.com/hashicorp/consul/agent
agent/testagent.go:32:2: testutil redeclared as imported package name
	previous declaration at agent/testagent.go:19:2
```
Go version 1.14.15.

--- a/agent/health_endpoint_test.go
+++ b/agent/health_endpoint_test.go
@@ -8,6 +8,7 @@ import (
 	"net/http/httptest"
 	"net/url"
 	"reflect"
+	"strconv"
 	"testing"
 	"time"
 

--- a/agent/testagent.go
+++ b/agent/testagent.go
@@ -16,7 +16,6 @@ import (
 	"time"
 
 	metrics "github.com/armon/go-metrics"
-	"github.com/hashicorp/consul/sdk/testutil"
 	"github.com/hashicorp/errwrap"
 	"github.com/hashicorp/go-hclog"
 	uuid "github.com/hashicorp/go-uuid"

